Scruffy.Commands/SlashCommands/GamesSlashCommandModule.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Discord.Interactions;
+
+using Scruffy.Services.Discord;
+using Scruffy.Services.Games;
+
+namespace Scruffy.Commands.SlashCommands;
+
+/// <summary>
+/// Games
+/// </summary>
+public class GamesSlashCommandModule : SlashCommandModuleBase
+{
+    #region Enums
+
+    /// <summary>
+    /// The type of game
+    /// </summary>
+    public enum GameType
+    {
+        Counter,
+        WordChain
+    }
+
+    /// <summary>
+    /// The action to perform on a certain game
+    /// </summary>
+    public enum GameCommandAction
+    {
+        Add,
+        Remove
+    }
+
+    #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// Counter game service
+    /// </summary>
+    public CounterGameService CounterGameService { get; set; }
+
+    /// <summary>
+    /// Word chain game service
+    /// </summary>
+    public WordChainGameService WordChainGameService { get; set; }
+
+    #endregion // Properties
+
+    #region Methods
+
+    /// <summary>
+    /// Adds the counter game to the channel
+    /// </summary>
+    /// <param name="type">The type of the mini game to manage.</param>
+    /// <param name="action">The action to perform on the mini game.</param>
+    /// <returns>A <see cref="Task"/> representing the asynchronous operation</returns>
+    [SlashCommand("games", "Manage mini games")]
+    [RequireContext(ContextType.Guild)]
+    //[RequireAdministratorPermissions]
```

Comment:
You can use `[DefaultPermission(false)]` to disable the command by default. Then we can enable the command only for administrators.